### PR TITLE
[Fix](row store) Real default value should be used instead of default…

### PR DESCRIPTION
**Pull request by eldenmoon**
… value in IColumn

Before this PR the default value is not correct, we should use default value in Frontend schema
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by eldenmoon** [comment]
run buildall

File: be/src/vec/jsonb/serialize.cpp [hunk 1]
```diff
@@ -98,11 +102,20 @@ void JsonbSerializeUtil::jsonb_to_block(const DataTypeSerDeSPtrs& serdes, const
...
                 DCHECK(col->size() == num_rows);
-                col->insert_default();
+                if (default_values[i].empty()) {
+                    col->insert_default();
+                } else {
+                    // col->insert_default();
+                    ReadBuffer value(
+                            reinterpret_cast<const unsigned char*>(default_values[i].data()),
+                            default_values[i].size());
+                    static_cast<void>(column_type_name.type->from_string(value, col.get()));
```
Comment (xiaokang) [inline be/src/vec/jsonb/serialize.cpp:117]: I remember that from_string is deprecated by amorynan.

File: regression-test/suites/point_query_p0/test_point_query.groovy [hunk 2]
```diff
@@ -188,10 +188,10 @@ suite("test_point_query") {
       qe_point_select stmt
       qe_point_select stmt
 
-      // sql """
-      //   ALTER table ${tableName} ADD COLUMN new_column1 INT default "0";
-      // """
-      // qe_point_select stmt 
+      sql """
+        ALTER table ${tableName} ADD COLUMN new_column1 INT default "0";
```
[the remark this answers is not in the document]
Comment (eldenmoon) [reply regression-test/suites/point_query_p0/test_point_query.groovy:192]: it's not deleted, but used

File: gensrc/thrift/Descriptors.thrift [hunk 3]
```diff
@@ -62,6 +62,7 @@ struct TSlotDescriptor {
   14: optional bool is_auto_increment = false;
   // subcolumn path info list for semi structure column(variant)
   15: optional list<string> column_paths
+  16: optional string col_default_value
```
[the remark this answers is not in the document]
Comment (eldenmoon) [reply gensrc/thrift/Descriptors.thrift:65]: variant fields is used which is `column_paths `

File: be/src/vec/jsonb/serialize.cpp (hunk 1 above)
Comment (eldenmoon) [reply be/src/vec/jsonb/serialize.cpp:117]: from_string is still used in many codes

**Comment by eldenmoon** [comment]
run buildall

**Comment by eldenmoon** [comment]
run buildall

**Comment by eldenmoon** [comment]
run buildall

**Comment by eldenmoon** [comment]
run buildall

File: be/src/vec/jsonb/serialize.cpp (hunk 1 above)
Comment (eldenmoon) [reply be/src/vec/jsonb/serialize.cpp]: done

**Review by xiaokang — APPROVED** [review]
LGTM

**Review by qidaye — APPROVED** [review]
LGTM

**Pull request merged**